modules/ROOT/pages/_partials/symbolicate-ios.adoc

```diff
@@ -0,0 +1,79 @@
+== Symbolication
+
+In order to make sense of a crash report you will need to symbolicate
+ its content.
+Symbolication being the process of replacing memory addresses in a crash report with human-readable function names and line numbers.
+To do this you need to have debug symbol (`.dSYM`) files for the application and for Couchbase Lite's framework.
+
+== Finding local dSYM
```

Comment:
I don't think this section is needed as it's for the application dSYM file (not CouchbaseLite dSYM file).